packages/components/Divider/SPEC.md

```diff
@@ -12,8 +12,18 @@ Content, text or an icon are also not supported on mobile platforms.
 
 ## Sample Code
 
+Add the Divider package to your project.
```

Comment:
I think this section is more for example usages, see https://github.com/microsoft/fluentui-react-native/blob/main/packages/components/Button/SPEC.md